### ISSUE TITLE
Collision issues (walking through blocks)

### ISSUE THREAD

**Issue by mpaoloni** [fogleman/Craft#209]
It seems like the collision system has some issues. I am able to walk (literally) into blocks by going towards corners at a perfect angle.

Tested with 1 and 2 blocks (one stacked on top the other).

2 blocks:
![2 blocks](https://i.imgur.com/JZvJ6iS.png)

1 block:
![1 block](https://i.imgur.com/FdqGuuy.png)

**Comment by twetzel59** [comment]
@mpaoloni Hmm, odd! Is this with fly on or off?

**Comment by twetzel59** [comment]
Yikes, just reproduced! I wonder how hard it is to fix.

**Comment by KcTec90** [comment]
Yikes got it too :p 

https://gyazo.com/6cd0b27864a9c7517d121d7b479e6027

https://youtu.be/Ir7Nahd-sAg

**Comment by ihalseide** [comment]
Yes, PR #216 fixes this. I also made a fix commit in my fork that uses a different method (swept AABB) for collision detection.
